dockerfiles/init/modules/keycloak/templates/che-realm.json.erb

```diff
@@ -58,8 +58,13 @@
     "enabled" : true,
     "clientAuthenticatorType" : "client-secret",
     "secret" : "56abc663-1c41-4ab2-81d1-d4b704249221",
+<% if scope.lookupvar('che::che_port') == '80' or scope.lookupvar('che::che_port') == '443' -%>
```

Comment:
Is there a chance that this addition will break deployment to openshift [here](https://github.com/eclipse/che/blob/master/dockerfiles/keycloak/kc_realm_user.sh#L23)? @eivantsov WDYT?

Reply:
It will, since currently I sed it run to replace puppet stuff with an actual URL - https://github.com/eclipse/che/blob/master/dockerfiles/keycloak/kc_realm_user.sh#L24 Since we use templates directly in Keycloak entrypoint, all puppet syntax needs to be removed there.

@hkolvenbach will you add those changes to Keycloak script?